### PR TITLE
Update zeropp.md

### PR DESCRIPTION
**Pull request by goodship1**
Doc corrections

### CODE REVIEW

**Review by loadams — APPROVED** [review]

File: docs/_tutorials/zeropp.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ We recommend that you read the tutorials on [Getting Started](/getting-started/)
...
- - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation
- - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient averaging.
+ - *qwZ* applies block-based quantization to reduce ZeRO parameter all-gather communication volume by half from FP16 to INT8.
+ - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation.
+ - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient. averaging.
 
-Collectively, the three optimization reduces communication volume by 4x compared to ZeRO baseline. Each of the three components can be enabled independent of each other and collectively as a group as described in the next section.
+Collectively, the three optimization reduces communication volume by 4x compared to ZeRO baseline. Each of the three. components can be enabled independent of each other and collectively as a group as described in the next section.
```
Comment (loadams) [inline docs/_tutorials/zeropp.md:17]: The period mid-sentence isn't needed.

**Review by loadams — CHANGES_REQUESTED** [review]

File: docs/_tutorials/zeropp.md [hunk 2]
```diff
@@ -10,11 +10,11 @@ We recommend that you read the tutorials on [Getting Started](/getting-started/)
 
 ## Three Components of ZeRO++
 ZeRO++ consists of three key designs, namely quantized weights (*qwZ*), hiearchical partitioning ZeRO (*hpZ*), and quantized gradients (*qgZ*):
- - *qwZ* applies block-based quantization to reduce ZeRO parameter all-gather communication volume by half from FP16 to INT8)
- - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation
- - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient averaging.
+ - *qwZ* applies block-based quantization to reduce ZeRO parameter all-gather communication volume by half from FP16 to INT8.
+ - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation.
+ - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient. averaging.
```
Comment (loadams) [inline docs/_tutorials/zeropp.md:15]: Period not needed after gradient

**Review by loadams — CHANGES_REQUESTED** [review]

File: docs/_tutorials/zeropp.md (hunk 2 above)
Comment (mrwyattii) [inline docs/_tutorials/zeropp.md:15]:
```suggestion
 - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient averaging.
```

File: docs/_tutorials/zeropp.md [hunk 3]
```diff
@@ -10,11 +10,11 @@ We recommend that you read the tutorials on [Getting Started](/getting-started/)
...
- - *qwZ* applies block-based quantization to reduce ZeRO parameter all-gather communication volume by half from FP16 to INT8)
- - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation
+ - *qwZ* applies block-based quantization to reduce ZeRO parameter all-gather communication volume by half from FP16 to INT8.
+ - *hpZ* eliminates inter-node backward parameter all-gather communication through data remapping and recomputation.
  - *qgZ* replaces gradients allreduce collective with a new communication efficient all-to-all based quantized gradient averaging.
 
-Collectively, the three optimization reduces communication volume by 4x compared to ZeRO baseline. Each of the three components can be enabled independent of each other and collectively as a group as described in the next section.
+Collectively, the three optimization reduces communication volume by 4x compared to ZeRO baseline. Each of the three. components can be enabled independent of each other and collectively as a group as described in the next section.
```
Comment (mrwyattii) [inline docs/_tutorials/zeropp.md:17]:
```suggestion
Collectively, the three optimization reduces communication volume by 4x compared to ZeRO baseline. Each of the three components can be enabled independent of each other and collectively as a group as described in the next section.
```

**Review by mrwyattii — APPROVED** [review]

**Pull request merged**